spec/models/client_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Client, type: :model do
+  it { should have_db_column(:first_name).of_type(:string) }
+  it { should have_db_column(:last_name).of_type(:string) }
+  it { should have_db_column(:website).of_type(:string) }
+  it { should have_db_column(:source).of_type(:integer) }
+  it { should have_db_column(:email).of_type(:string) }
+  it { should have_db_column(:location).of_type(:string) }
+  it { should have_many(:projects) }
+
+  it "valid if email is available" do
```

Comment:
We want to test if email or any other required attribute is missing it raise an error

Reply:
fixed